src/Raven.Studio/typescript/models/wizard/nodeInfo.ts

```diff
@@ -41,6 +42,11 @@ class nodeInfo {
             return this.nodeTag() === 'A';
         });
         
+        this.isLoopbackOnly = ko.pureComputed(() => {
+            const ips = this.ips();
+            return ips.length === 1 && ips[0].ip() === "127.0.0.1";
```

Comment:
Also, `localhost`
And in general, anything that is `127.x.x.x`

Reply:
ok I will change that in few places.